src/window.rs

```diff
@@ -524,25 +524,25 @@ impl Window {
         self.window.scale_factor()
     }
 
-    /// Emits a [`Event::RedrawRequested`] event in the associated event loop after all OS
-    /// events have been processed by the event loop.
+    /// Requests a future [`Event::RedrawRequested`] event to be emitted in a way that is
+    /// synchronized and / or throttled by the window system.
```

Comment:
```suggestion
    /// synchronized and / or throttled by the windowing system.
```

Reply:
yeah I suppose either is ok but not sure atm if 'windowing' is used more consistently perhaps. I'll at least also use 'windowing system' in the paragraph below for consistency